src/NServiceBus.Core/Persistence/Development/SagaPersister/DevelopmentSagaPersisterConfigExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace NServiceBus
+{
+    using Features;
+    using Persistence;
+
+    /// <summary>
+    /// Configuration options for the development saga persister.
+    /// </summary>
+    public static class DevelopmentSagaPersisterConfigExtensions
+    {
+        /// <summary>
+        /// Configures the location where sagas are stored.
+        /// </summary>
+        /// <param name="config">Config object to extend.</param>
+        /// <param name="path">The storage path.</param>
+        public static void SagaStorageDirectory(this PersistenceExtensions<DevelopmentPersistence, StorageType.Sagas> config, string path)
```

Comment:
I think you should use the following signature: `public static void SagaStorageDirectory(this PersistenceExtensions<DevelopmentPersistence> config, string path)` instead (no storage type)
as you may also want to configure the directory when using `UsePersistence<DevelopmentPersistence>()` (we did the same for `UsePersistence<MsmqPersistence>().SubscriptionQueue(...)`)